cmd/root.go

```diff
@@ -45,6 +45,9 @@ func newRootCmd() *cobra.Command {
 	rootCmd.PersistentFlags().StringVarP(&(config.GetConfig().Project), "project", "p", "", "Specifies the Flyte project.")
 	rootCmd.PersistentFlags().StringVarP(&(config.GetConfig().Domain), "domain", "d", "", "Specifies the Flyte project's domain.")
 	rootCmd.PersistentFlags().StringVarP(&(config.GetConfig().Output), "output", "o", printer.OutputFormatTABLE.String(), fmt.Sprintf("Specifies the output type - supported formats %s", printer.OutputFormats()))
+	rootCmd.PersistentFlags().StringVarP(&(config.GetConfig().Filters), "filters", "", "", "Specifies the Filters")
+	rootCmd.PersistentFlags().StringVarP(&(config.GetConfig().SortBy), "sort-by", "s", "", "Specifies the Sort-by")
+	rootCmd.PersistentFlags().Int32Var(&(config.GetConfig().Limit), "limit", 100, "Specifies the limit on results")
```

Comment:
nit: (you will need to make defaultConfig public or something...)
```suggestion
	rootCmd.PersistentFlags().Int32Var(&(config.GetConfig().Limit), "limit", config.defaultConfig.Limit, "Specifies the limit on results")
```